### ISSUE TITLE
Parser raises an exception for not supported query type when query starts with a comment

### ISSUE THREAD

**Issue by macbre** [macbre/sql-metadata#163]
When working on [index-digest's PR](https://github.com/macbre/index-digest/pull/244) I have found the following issue.

An example call:

```python
tables = get_query_tables("/* foo */ INSERT IGNORE INTO `0070_insert_ignore` VALUES (9, '123', '2017-01-01');")
```

Error raised:

```
/* foo */ INSERT IGNORE INTO `0070_insert_ignore` VALUES (9, '123', '2017-01-01');

        if self._query_type == "UNSUPPORTED":
>           raise ValueError("Not supported query type!")
E           ValueError: Not supported query type!

env/lib/python3.9/site-packages/sql_metadata/parser.py:94: ValueError
```

Additionally, we should also log the query itself that caused this exception to be raised.

**Comment by collerek** [comment]
We should probably remove the comments before the check. Didn't anticipate that but that's actually quite common, some kind of explanation what the query is doing.

You mean you want to print the whole query in exception? The query can be huge (i.e. stored procedure declaration).

**Comment by macbre** [comment]
> You mean you want to print the whole query in exception? The query can be huge (i.e. stored procedure declaration).

I'd rather log it via `logger` module.

**Comment by macbre** [comment]
Once #164 is merged I will publish v2.2.1 with this fix.

**Comment by collerek** [comment]
Can you check also #162 - it's small and actually simplifies handling of with clauses?
